python/noteable/git-wrapper.sh

```diff
@@ -5,11 +5,15 @@
 # Allowed command list
 allowed_commands=( "commit" "pull" "push" "status" "diff" "add" "fetch" "log" "version" )
 
-# Check if the command is allowed
-# shellcheck disable=SC2076
-if [[ ! " ${allowed_commands[*]} " =~ " ${1} " ]]; then
-    echo "That git command is not allowed, contact support@noteable.io if you think this is a mistake."
-    exit 1
+# If the NTBL_IS_GIT_BACKED env var is "true" then we check to see if the command is allowed.
+# If the env var is not set or is not "true" then we just pass through to git.
+if [[ "${NTBL_IS_GIT_BACKED}" == "true" ]]; then
```

Comment:
looks like this was a bit behind; noticed during build errors installing `dev_packages` where a `git clone` was triggered